openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1562,7 +1442,7 @@ static const char* authentication_method(const SSL* ssl) {
                 // No cipher available so return UNKNOWN.
                 return UNKNOWN_AUTH_METHOD;
             }
-            return cipher_authentication_method(sk_value((_STACK*) ciphers, 0));
+            return SSL_cipher_authentication_method(sk_value((_STACK*) ciphers, 0));
```

Comment:
can we clean this up as a followup PR, or are you concerned about API breaking changes?